CODE/Sources/PennyBOT/Handlers/+String.swift

```diff
@@ -0,0 +1,22 @@
+
+extension String {
+    func foldForPingCommand() -> String {
```

Comment:
just manipulates the string to a form more appropriate for auto-ping. Makes it lowercased and removes leading/trailing spaces